['Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpon0e371h/solution_0.py", line 67, in <module>\n    print(Solution().kthCharacter(k = 7, operations = [1,1,1]))\nTypeError: kthCharacter() got an unexpected keyword argument \'k\'\n', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpon0e371h/solution_1.py", line 67, in <module>\n    print(Solution().kthCharacter(k = 20, operations = [0,0,0,1,1]))\nTypeError: kthCharacter() got an unexpected keyword argument \'k\'\n']